packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -311,7 +311,16 @@ const DMT5CompareModalContent = ({
     }, []);
 
     const getAvailableAccountsContent = (_content: TModalContentProps[]) => {
-        if (!is_logged_in) return _content;
+        if (!is_logged_in) {
+            if (show_eu_related) {
+                return _content;
+            }
+            const mt5_data = _content.map(item => {
+                const { derivx, ...rest } = item.values; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we need to restructure the object?

Reply:
@matin-binary its because `item.values` have derivx related content which we dont want to show in dmt5 dashboard.